src/style.css

```diff
@@ -1 +1,35 @@
 /* add styles here */
+
+/* okay */
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+  align-items: center;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background-image: url('./images/star.svg');
+  background-position: center;
+  background-repeat: no-repeat;
+}
```

Comment:
The `.stars__star` class is descriptive and follows BEM naming conventions, which is great for understanding the relationship between elements and their parent block.